rosenpass/src/protocol.rs

```diff
@@ -584,6 +685,8 @@ impl Peer {
             session: None,
             handshake: None,
             initiation_requested: false,
+            cookie_tau: CookieSecret::None,
+            last_sent_mac: None,
```

Comment:
This information should already be present in the retransmission buffer…why don't we fetch it from there?

Reply:
done